lib/multiply.rb

```diff
@@ -1,2 +1,9 @@
 def multiply(num1, num2)
-end
\ No newline at end of file
+output = []
+num1.times do
+    output << "#{num2}"
```

Comment:
`"#{num2}"` makes `num2` a string. Does it need to be a string?

Did you consider `output << num2` as is?